src/writeData/components/WriteDataDetailsContext.tsx

```diff
@@ -80,7 +80,7 @@ const WriteDataDetailsContextProvider: FC<Props> = ({
 }
 
 const mstp = (state: AppState) => {
-  const buckets = getAll<Bucket>(state, ResourceType.Buckets)
+  const buckets = getAll<Bucket>(state, ResourceType.Buckets) ?? []
```

Comment:
Just curious - Should this be something thats done here, or in the getAll response so as to potentially guard against this elsewhere? Or does the function response necessitate potentially returning a null value?

Reply:
Good point. moved it up the pipeline